includes/menu/menu_config.php

```diff
@@ -93,4 +93,18 @@
 //after all other menus
 $menuExtras[7] = ['item', 'home.png', '', 'http://www.home.com', '_blank']
 */
+
+/*
+In multiuser calendar, this function print only the personal agenda of the day with event's description and place
+*/
+$base_url = "http://" . $_SERVER['HTTP_HOST'] . "/day.php?friendly=1";
```

Comment:
Should not be assuming http here in case the site is using https.
```suggestion
$base_url = ( empty($HTTPS) ? "http://" : "https://" ) . $_SERVER['HTTP_HOST'] . "/day.php?friendly=1";
```